### ISSUE TITLE
Error when using read_abs

### ISSUE THREAD

**Issue by eddyzhang2018** [MattCowgill/readabs#64]
Hi Matt,

After I loaded the readabs library and run read_abs(), it returns the following error:

> > read_abs(cat_no = "6202.0", table = 1)
Finding filenames for tables from ABS catalogue 6202.0
Unknown IO errorfailed to load external entity "http://ausstats.abs.gov.au/servlet/TSSearchServlet?catno=6202.0&ttitle=1&pg=1"
Error: 1: Unknown IO error2: failed to load external entity "http://ausstats.abs.gov.au/servlet/TSSearchServlet?catno=6202.0&ttitle=1&pg=1"

Is this an error due to my R or something changed to ABS website?

Thank you,
Eddy

**Comment by MattCowgill** [comment]
Hi Eddy,
I can't reproduce your problem. The code works for me. Here is a reprex: 

``` r
library(readabs)
#> Environment variable 'R_READABS_PATH' is unset. Downloaded files will be saved in a temporary directory.
#> You can set 'R_READABS_PATH' at any time. To set it for the rest of this session, use
#>  Sys.setenv(R_READABS_PATH = <path>)

lfs <- read_abs(cat_no = "6202.0", tables = 1)
#> Finding filenames for tables corresponding to ABS catalogue 6202.0
#> Attempting to download files from catalogue 6202.0, Labour Force, Australia
#> Extracting data from downloaded spreadsheets
#> Tidying data from imported ABS spreadsheets

lfs
#> # A tibble: 57,228 x 12
#>    table_no sheet_no table_title date       series value series_type data_type
#>    <chr>    <chr>    <chr>       <date>     <chr>  <dbl> <chr>       <chr>    
#>  1 6202001  Data1    Table 1. L… 1978-02-01 Emplo… 6008. Trend       STOCK    
#>  2 6202001  Data1    Table 1. L… 1978-03-01 Emplo… 6015. Trend       STOCK    
#>  3 6202001  Data1    Table 1. L… 1978-04-01 Emplo… 6022. Trend       STOCK    
#>  4 6202001  Data1    Table 1. L… 1978-05-01 Emplo… 6027. Trend       STOCK    
#>  5 6202001  Data1    Table 1. L… 1978-06-01 Emplo… 6031. Trend       STOCK    
#>  6 6202001  Data1    Table 1. L… 1978-07-01 Emplo… 6033. Trend       STOCK    
#>  7 6202001  Data1    Table 1. L… 1978-08-01 Emplo… 6032. Trend       STOCK    
#>  8 6202001  Data1    Table 1. L… 1978-09-01 Emplo… 6033. Trend       STOCK    
#>  9 6202001  Data1    Table 1. L… 1978-10-01 Emplo… 6035. Trend       STOCK    
#> 10 6202001  Data1    Table 1. L… 1978-11-01 Emplo… 6040. Trend       STOCK    
#> # … with 57,218 more rows, and 4 more variables: collection_month <chr>,
#> #   frequency <chr>, series_id <chr>, unit <chr>
```

<sup>Created on 2020-01-23 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

Can you please try running this command:
```r
curl::has_internet()
```

If the result is `FALSE`, your R session cannot connect to the internet. This will need to be resolved before you can use `read_abs()`. 

If the result is `TRUE`, please try running this code:

```r
test_abs_robots <- function() {
  tmp <- tempfile()
  on.exit(unlink(tmp, recursive = TRUE), add = TRUE)
  result <- tryCatch({
    suppressWarnings(utils::download.file(
      "https://www.abs.gov.au/robots.txt",
      destfile = tmp,
      quiet = TRUE))
    file.exists(tmp)
  },
  error = function(e) {
    FALSE
  }
  )
  return(result)
}

test_abs_robots()
```

If the result is `TRUE` then your R session can connect to the internet in general, and the ABS in particular, so I am not sure what the problem is given that I cannot reproduce the error. In this case, can you please try restarting R, execute the code again, then tell me the results?

If the result of `test_abs_robots()` is `FALSE` then your R session can access the internet, but not the ABS website.

**Comment by eddyzhang2018** [comment]
Hi Matt,

Thanks for your detailed reply.

I have tried the test you mentioned above. All returned TRUE. So, I tested the code in R base (was using RStudio). And the read_abs() works well. In addition, tested on Rstudio cloud the function works well.

I uninstalled my Rstudio and R base on my computer and installed latest version. It works now. I doubt it might be some problem with my older version of Rstudio.

Thank you,
Eddy
